packages/table/test/tableTests.tsx

```diff
@@ -56,6 +56,24 @@ describe("<Table>", () => {
         expect(table.find(".bp-table-column-headers .bp-table-header", 1).element).to.be.ok;
     });
 
+    it("Renders correctly with loading options", () => {
+        const renderCell = () => <Cell>my cell value</Cell>;
+        const loadingOptions = [
+            TableLoadingOption.CELLS,
+            TableLoadingOption.COLUMN_HEADERS,
+            TableLoadingOption.ROW_HEADERS,
+        ];
+        const tableHarness = harness.mount(
+            <Table loadingOptions={loadingOptions} numRows={2}>
+                <Column name="Column0" renderCell={renderCell} />
+                <Column name="Column1" renderCell={renderCell} />
+            </Table>,
+        );
+        const tableElement = tableHarness.element.children[0];
+
+        expect(tableElement.textContent).to.equal("");
```

Comment:
👎  but it doesn't test that they __render loading content__. it tests that they __don't render their text content__, which is not _necessarily_ the same thing. do you see the difference?